src/main/java/io/microconfig/osdf/state/OpenShiftCredentials.java

```diff
@@ -0,0 +1,30 @@
+package io.microconfig.osdf.state;
+
+import lombok.*;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@EqualsAndHashCode
+public class OpenShiftCredentials {
```

Comment:
do not copy logic from Credentials class

Reply:
Has been changed